metpy/calc/indices.py

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2008-2015 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Contains calculation of various derived indicies."""
+
+from ..calc import mixing_ratio, saturation_vapor_pressure
+from ..calc.tools import get_layer
+from ..constants import g, rho_l
+from ..package_tools import Exporter
+from ..units import units
+
+exporter = Exporter(globals())
+
+
+def precipitable_water(dewpt, p, top=400 * units('hPa')):
+    # Implementation of integral calculation based partly on SRH integral
+    # approximation, as found in Markowski and Richardson (2013) or in
+    # SharpPy
+    r"""Calculate precipitable water through the depth of a sounding.
+
+    Default layer depth is sfc-400 hPa. Formula used is from the AMS glossary:
```

Comment:
At the very least we'd need to add a reference to AMS glossary--but we can't do better?